modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -612,6 +612,43 @@ private void SetDiagonal(Vector3 diagonal)
             return b;
         }
 
+        /// <summary>
+        /// Creates a <see cref="Basis"/> with a rotation such that the forward
+        /// axis (-Z) points towards the <paramref name="target"/> position.
+        /// The up axis (+Y) points as close to the <paramref name="up"/> vector
+        /// as possible while staying perpendicular to the forward axis.
+        /// The resulting Basis is orthonormalized.
+        /// The <paramref name="target"/> and <paramref name="up"/> vectors
+        /// cannot be zero, and cannot be parallel to each other.
+        /// </summary>
+        /// <param name="target">The position to look at.</param>
+        /// <param name="up">The relative up direction.</param>
+        /// <returns>The resulting basis matrix.</returns>
+        public static Basis LookingAt(Vector3 target, Vector3 up)
+        {
+#if DEBUG
+            if (target.IsZeroApprox())
+            {
+                throw new ArgumentException("The vector can't be zero.", nameof(target));
+            }
+            if (up.IsZeroApprox())
+            {
+                throw new ArgumentException("The vector can't be zero.", nameof(up));
+            }
+#endif
+            Vector3 column2 = -target.Normalized();
+            Vector3 column0 = up.Cross(column2);
```

Comment:
I'm not sure what the advantage is of not just calling these `x`, `y`, and `z` like in core.

Reply:
It's likely because these are the names of the parameters in the `Basis(Vector3, Vector3, Vector3)` constructor.